build.py

```diff
@@ -187,13 +187,11 @@ def core_cmake_args(components, backends, install_dir):
                      ('tensorflow2' in backends))))
 
     for be in (CORE_BACKENDS + NONCORE_BACKENDS):
-        if not be.startswith('tensorflow'):
+        if not (be.startswith('tensorflow') or be == 'dali'):
```

Comment:
Why adding dali? Won't that not include dali in any case?

Reply:
Fixed